### PR TITLE
 Added user model

### PR DESCRIPTION
**Pull request by harshareddy794**
Lined issue #19

### CODE REVIEW

File: server/Models/user.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose')
+
+const userSchema = new mongoose.Schema({
+
+    name : String,
+    phoneNo : Number,
```
Comment (BhavyaaArora-08) [inline server/Models/user.js:9]: name should be required

File: server/Models/user.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    email : {
+        type: String, 
+        unique: true, 
+        required: true
+    },
+    Address : String,
+    password : String,
+    token : {
+        type: String, 
+        unique: true,
```
Comment (BhavyaaArora-08) [inline server/Models/user.js]: the token assigned to the user will always be unique, so no need to add that.

**Comment by harshareddy794** [comment]
Made changes have a look @Codess08

**Comment by Codess08** [comment]
Also, Have a look at this
image

This will the folder structure of our application
So, npm init in root directory and not in server folder.

**Comment by Codess08** [comment]
@harshareddy794 I think you didn't attend the meeting right?
I didn't merge your PR because I have created a file structure you have to create files according to that only.
Please have a look, else your code is fine.

**Comment by BhavyaaArora-08** [comment]
When you initialized npm, did you not set main to server.js?
You simply remove package.json file I have already created one.

**Pull request closed without merging**